src/ProductConstructionService/ProductConstructionService.Api/Telemetry/TelemetryRecorder.cs

```diff
@@ -17,6 +17,11 @@ public ITelemetryScope RecordJob(Job job)
         return new TelemetryScope($"JobExecuted", _logger, _telemetryClient, new() { ["JobType"] = job.Type }, []);
     }
 
+    public ITelemetryScope RecordGitClone(string repo)
+    {
+        return new TelemetryScope($"GitClone", _logger, _telemetryClient, new() { ["Repo"] = repo }, []);   
```

Comment:
Maybe this is more clear what it is? Repo could be a name of the repo only
```suggestion
        return new TelemetryScope($"GitClone", _logger, _telemetryClient, new() { ["Uri"] = repo }, []);   
```

Reply:
yup, fixed